static/js/compose_validate.js

```diff
@@ -593,6 +593,46 @@ export function check_overflow_text() {
     return text.length;
 }
 
+export function edit_check_overflow_text() {
```

Comment:
We should probably refactor the `check_overflow_text` with parameters for which area to operate on, so that it can be used in edit state as well 🤔

Reply:
Ok. I'll make the changes.